### PR TITLE
feat(view-mode): add view-mode button and modal

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
- add view-mode button to widget

스크린샷 2023-04-26 오후 12 33 28

- show view-mode modal
스크린샷 2023-04-26 오후 12 33 39

### CODE REVIEW

**Review by WANZARGEN — APPROVED** [review]
LGTM 👍 
I left a very tiny suggestion!

File: apps/web/src/services/dashboards/widgets/_components/WidgetViewModeModal.vue [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+             class="modal-header"
+        >
+            <p-heading :title="state.widgetInfo?.title ?? ''"
+                       show-back-button
+                       @click-back-button="handleCloseModal"
+            />
+            <p-icon-button name="ic_close"
+                           class="close-button"
+                           @click="handleCloseModal"
+            />
```
[the remark this answers is not in the document]
Comment (piggggggggy) [reply apps/web/src/services/dashboards/widgets/_components/WidgetViewModeModal.vue:15]: I agree

**Review by piggggggggy — APPROVED** [review]
Loog great!! I agree handling long title case @WANZARGEN said.

File: apps/web/src/services/dashboards/widgets/_components/WidgetViewModeModal.vue [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    @apply bg-white;
+    display: none;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 100;
+    transition: all 0.2s ease-in-out;
+    transform: scale(1);
```
[the remark this answers is not in the document]
Comment (yuda110) [reply apps/web/src/services/dashboards/widgets/_components/WidgetViewModeModal.vue:96]: It doesn't work properly yet. 😅

File: apps/web/src/services/dashboards/widgets/_components/WidgetFrame.vue [hunk 3]
```diff
@@ -229,23 +246,36 @@ const handleDeleteModalConfirm = () => {
...
 
     .widget-header {
-        @apply flex justify-between items-center;
+        @apply flex items-center;
         .title {
+            @apply truncate;
             overflow: hidden;
             text-overflow: ellipsis;
-            display: -webkit-box;
+            display: block;
```
[the remark this answers is not in the document]
Comment (yuda110) [reply apps/web/src/services/dashboards/widgets/_components/WidgetFrame.vue:254]: Title changed to 1 line & ellipsis!

before
스크린샷 2023-04-27 오전 10 54 40

after
스크린샷 2023-04-27 오전 10 54 21

File: apps/web/src/services/dashboards/widgets/_components/WidgetViewModeModal.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (yuda110) [reply apps/web/src/services/dashboards/widgets/_components/WidgetViewModeModal.vue:15]: Thanks! I'll check that when I markup modal.

**Pull request merged**